stack-cmbm/recipes/deploy-nginxapp.rb

```diff
@@ -20,6 +20,8 @@
     next unless allow_deploy(app_name, 'cm', 'nginxapp_cm')
   when 'bm'
     next unless allow_deploy(app_name, 'bm', 'nginxapp_bm')
+  when 'cmbm'
```

Comment:
I would probably remove this again.

Brian split the apps today. And along this pattern, I would probably also do "cm" and "bm" to make local reflect playground/production.